arm-storage/2016-05-01/swagger/storage.json

```diff
@@ -0,0 +1,1327 @@
+          "enum": [
+            "s",
+            "c",
+            "o"
+          ],
+          "x-ms-client-name": "ResourceTypes",
+          "description": "Sets the signed resource types that are accessible with the account SAS."
+        },
+        "signedPermission": {
+          "type": "string",
+          "enum": [
+            "r",
+            "d",
+            "w",
+            "l",
+            "a",
+            "c",
+            "u",
+            "p"
+          ],
+          "x-ms-client-name": "Permissions",
+          "description": "Sets the signed permissions for the account SAS."
+        },
+        "signedIp": {
+          "type": "string",
+          "x-ms-client-name": "IPAddressOrRange",
+          "description": "Sets an IP address or a range of IP addresses from which to accept requests."
+        },
+        "signedProtocol": {
+          "type": "string",
+          "enum": [
+            "https,http",
+            "https"
+          ],
+          "x-ms-enum": {
+            "name": "HttpProtocol",
+            "modelAsString": false
+          },
+          "x-ms-client-name": "Protocols",
+          "description": "Sets the protocol permitted for a request made with the account SAS."
+        },
+        "signedStart": {
+          "type": "string",
+          "format": "date-time",
+          "x-ms-client-name": "SharedAccessStartTime",
+          "description": "Sets the time at which the SAS becomes valid."
+        },
+        "signedExpiry": {
+          "type": "string",
+          "format": "date-time",
+          "x-ms-client-name": "SharedAccessExpiryTime",
+          "description": "Sets the time at which the shared access signature becomes invalid."
+        },
+        "keyToSign": {
+          "type": "string",
+          "description": "Sets the key to sign the account SAS token with."
+        }
+      },
+      "required": [
+        "signedServices",
+        "signedResourceTypes",
+        "signedPermission",
+        "signedExpiry"
+      ],
+      "description": "The parameters to list SAS credentials of a storage account."
+    },
+    "ListAccountSasResponse": {
+      "properties": {
+        "accountSasToken": {
+          "readOnly": true,
+          "type": "string",
+          "description": "List SAS credentials of storage account."
+        }
+      },
+      "description": "The List SAS credentials operation response."
+    },
+    "ServiceSasParameters": {
+      "properties": {
+        "canonicalizedResource": {
+          "type": "string",
+          "x-ms-client-name": "CanonicalizedResource",
+          "description" : "Sets the canonical path to the signed resource."
+        },
+        "signedResource": {
+          "type": "string",
+          "enum": [
+            "b",
+            "c",
+            "f",
+            "s"
+          ],
+          "x-ms-client-name": "Resource",
+          "description": "Sets the signed services accessible with the service SAS."
```

Comment:
General comment: Can you please describe these single letter enum values in their descriptions? That would be of great help to the customer.